app/controllers/course/lesson_plan_items_controller.rb

```diff
@@ -1,11 +1,16 @@
 class Course::LessonPlanItemsController < Course::ComponentController
   before_action :load_lesson_plan_items, only: [:index]
+  # This can only be done with Bullet once Rails supports polymorphic +inverse_of+.
+  around_action :without_bullet, only: [:index]
```

Comment:
...then legitimately getting n+1 queries from the lesson plan entries would not trigger bullet.